css-ui-4/Overview.bs

```diff
@@ -1324,6 +1322,7 @@ Applies to: all enabled elements
 Inherited: no
 Percentages: N/A
 Computed value: as specified
+Animation type: not animatable
```

Comment:
I think this one should be discrete. If you're animating the relative position of things, it seems reasonable to change the `nav-*` mapping half way through the animation to reflect the new locations. Animating discretely from one keyword to another doesn't seem to have any particular implementation difficulty either (no box to tear up or down halfway through the animation or any such shenanigans), so I don't see any reason to make this not animatable.

Reply:
Agree, I don't see a reason to make this non-animatable.